### PR TITLE
Staking NFT contract events

### PR DESCRIPTION
**Pull request by itzmeanjan**
Added `Transfer` event from StakingNFT contract, along with historical token ownership tracking capability

### CODE REVIEW

File: root/config/mainnet.json [hunk 1]
```diff
@@ -20,6 +20,10 @@
     "stakingInfo": {
```
Comment (jdkanani) [inline root/config/mainnet.json:20]: Please also change goerli.json and add respective NFT for the same.

**Review by jdkanani — APPROVED** [review]

File: root/config/mainnet.json (hunk 1 above)
Comment (jdkanani) [inline root/config/mainnet.json:20]: Please also change goerli.json and add respective NFT for the same.

**Review by jdkanani — CHANGES_REQUESTED** [review]

File: root/config/mainnet.json (hunk 1 above)
Comment (itzmeanjan) [reply root/config/mainnet.json:20]: Ok, I'll do so.

Comment (itzmeanjan) [reply root/config/mainnet.json:20]: Made changes.

**Review by jdkanani — APPROVED** [review]

**Pull request merged**